src/test/scala/cromwell/engine/SingleWorkflowRunnerActorSpec.scala

```diff
@@ -1,23 +1,20 @@
 package cromwell.engine
 
-import cromwell.HelloWorldActorSpec._
-import cromwell.util.SampleWdl.ThreeStep
-import akka.actor.ActorSystem
 import akka.testkit.EventFilter
-import com.typesafe.config.ConfigFactory
 import cromwell.CromwellTestkitSpec
+import cromwell.util.SampleWdl.ThreeStep
+
 import scala.language.postfixOps
 
-class SingleWorkflowRunnerActorSpec extends CromwellTestkitSpec(ActorSystem("ActorWorkflowManagerSpec", ConfigFactory.parseString(Config))) {
-  val actorSystem = super.getActorSystem
-  val workflowManagerActor = actorSystem.actorOf(WorkflowManagerActor.props)
+class SingleWorkflowRunnerActorSpec extends CromwellTestkitSpec(CromwellTestkitSpec.system("SingleWorkflowRunnerActorSpec")) {
```

Comment:
This invocation of `system` seems a bit awkward.  How about an alternate `CromwellTestkitSpec` constructor that takes just the actor system name?